modules/tensor_mechanics/test/tests/2D_different_planes/tests

```diff
@@ -22,12 +26,16 @@
                 Outputs/file_base=planestrain_xy_finite_out'
     exodiff = 'planestrain_xy_finite_out.e'
     abs_zero = 1e-7
+    requirement = "The tensor mechanics strain calculators shall solve plane strain in the x-y plane for finite strain"
+    issues = '#11257'
```

Comment:
@permcody as I commented on the other merge request I'm not including design here yet because those .md files are currently being added as part of another merge request. This only applies for the plane strain cases.